themes/hugo-planetarium/assets/scss/style.scss

```diff
@@ -43,6 +43,25 @@ blockquote p {
   display: inline;
 }
 
+blockquote > p:last-child {
+  cite:only-child { display: block; margin-top: 1em; }
+  cite:before { content: '\2014'; } // em dash
+  cite:lang(ko):before { content: '\2014\3008'; } // em dash and opening bracket
+}
+
+table {
+  border-collapse: collapse;
+
+  th, td {
+    border: 1px solid #ccc;
+    padding: 0.5em;
+
+    &:has(> strong:only-child) {
```

Comment:
The support state of `:has` isn't very good as of now: https://caniuse.com/css-has

Could we replace this with something like `<thead>`?

Reply:
Unfortunately it's not for table headers, but for highlighted columns.  As a Firefox user, I'm also disappointed that it's not properly rendered as I intended, I find it okay as the style expects markups to be semantic and those highlighted columns would look bold anyway.

Reply:
Yeah, it is a pity. I think it can be seen as a progressive enhancement until the support catches up.